src/phoenix/db/insertion/helpers.py

```diff
@@ -49,3 +57,149 @@ def insert_stmt(
             return stmt_sqlite.on_conflict_do_update(column_names, set_=set_)
         assert_never(on_conflict)
     assert_never(dialect)
+
+
+ADJECTIVES = [
+    "amazing",
+    "amusing",
+    "astonishing",
+    "awesome",
+    "breathtaking",
+    "captivating",
+    "charismatic",
+    "charming",
+    "curious",
+    "dazzling",
+    "delightful",
+    "dramatic",
+    "enchanting",
+    "entertaining",
+    "enthralling",
+    "exceptional",
+    "extraordinary",
+    "fantastic",
+    "fascinating",
+    "glamorous",
+    "impressive",
+    "incredible",
+    "inspiring",
+    "intriguing",
+    "magnificent",
+    "marvelous",
+    "mesmerizing",
+    "outstanding",
+    "phenomenal",
+    "remarkable",
+    "riveting",
+    "sensational",
+    "spectacular",
+    "spellbinding",
+    "splendid",
+    "stupendous",
+    "tantalizing",
+    "terrific",
+    "unforgettable",
+    "wonderful",
+]
+BIRD_SPECIES = [
+    "accipiter-soloensis",
+    "acestrura-bombus",
+    "aimophila-sumichrasti",
+    "anas-wyvilliana",
+    "apalis-sharpii",
+    "aplonis-mavornata",
+    "auriparus-flaviceps",
+    "aërornis-senex",
```

Comment:
Maybe nix this one since people won't be able to type it easily.

Reply:
good eye